ts/indicators/dpo.src.ts

```diff
@@ -123,15 +123,16 @@ H.seriesType<Highcharts.DPOIndicator>(
     {
         nameBase: 'DPO',
         getValues: function (
-            series: Highcharts.DPOIndicator,
+            series: Highcharts.Series,
             params: Highcharts.DPOIndicatorParamsOptions
         ): (boolean|Highcharts.IndicatorValuesObject) {
             var period: number = (params.period as any),
                 index: number = (params.index as any),
                 offset: number = Math.floor(period / 2 + 1),
                 range: number = period + offset,
                 xVal: Array<number> = series.xData || [],
-                yVal: (Array<number>|Array<Array<number>>) = series.yData || [],
+                yVal: (Array<number>|Array<Array<number>>) =
+                    (series.yData as any) || [],
```

Comment:
Additionally the `DPOIndicator.yData` property should be removed in the class declaration.